.github/workflows/sanity.yml

```diff
@@ -93,6 +93,16 @@ jobs:
         env:
           CQ_NO_TELEMETRY: 1
 
+      - name: Policy Run from config.hcl configuration
+        run: |
+          go run ./main.go policy run --config=internal/test/test_policy.hcl --disable-fetch-check &> policy_run.txt
```

Comment:
`policy run` fails if the policy tests failed, so this is the best way I could come up with to verify it didn't panic